addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -0,0 +1,38 @@
+# binding
+binding.kodi.name = Kodi Binding
+binding.kodi.description = Das Kodi Binding erlaubt das Kodi Media Center zu steuern 
+
+# thing types
+thing-type.kodi.kodi.label = Kodi Media Center
+thing-type.kodi.kodi.description = Kodi Media Center Binding
+thing-type.config.kodi.kodi.ipAddress.label = Netzwerkaddresse
+thing-type.config.kodi.kodi.ipAddress.description = IP Addresse oder Host Name des Kodi Media Centers
+thing-type.config.kodi.kodi.port.label = Port
+thing-type.config.kodi.kodi.port.description = Port des Web Socket Services des Kodi Media Centers
+
+# channel types
+channel-type.kodi.volume.label = Lautst�rke
+channel-type.kodi.volume.description = Lautst�rke des Kodi Media Centers
+channel-type.kodi.mute.label = Stumm schalten
+channel-type.kodi.mute.description = Das Kodi Media Centers stumm schalten
+channel-type.kodi.control.label = Kontrolle
+channel-type.kodi.control.description = Steuert das Kodi Media Center (z.B. Play, Pause, Stop)
+channel-type.kodi.playuri.label = URI abspielen
+channel-type.kodi.playuri.description = Spielt einen URI ab
+channel-type.kodi.shownotification.label = Nachricht anzeigen
+channel-type.kodi.shownotification.description = Zeigt eine Nachricht auf dem Bildschirm an
+channel-type.kodi.input.label = Tastendruck
+channel-type.kodi.input.description = Schickt einen Tastendruck an das Kodi Media Center um auf dem Bildschirm zu navigieren
+channel-type.kodi.inputtext.label = Texteingabe
+channel-type.kodi.inputtext.description = Schickt eine Zeichenkette als Eingabe an das Kodi Media Center
+channel-type.kodi.title.label = Titel
+channel-type.kodi.title.description = Titel des aktuellen St�cks oder Films
+channel-type.kodi.title.label = Titel
+channel-type.kodi.title.description = Titel des aktuellen St�cks oder Films
+channel-type.kodi.album.label = Album
+channel-type.kodi.album.description = Album des aktuellen St�cks
+channel-type.kodi.artist.label = K�nstler
```

Comment:
In the meantime I have created an issue in the Smarthome repository (https://github.com/eclipse/smarthome/issues/2639) because other bindings have the same problem.